etc/profile-a-l/gajim.profile

```diff
@@ -47,9 +49,17 @@ shell none
 tracelog
 
 disable-mnt
-private-bin bash,gajim,gajim-history-manager,gpg,gpg2,paplay,python,python3,sh,zsh
+private-bin bash,gajim,gajim-history-manager,paplay,python,python3,sh,zsh
```

Comment:
Has gajim removed gpg support?

Reply:
No but it comes with only plugin installer addon and gpg is a separate addon that needs to be installed unless the distro packages it as a dependency; if we are supporting every addon (or the more important ones like two gpg addons,omemo,indicator support etc.) the profile needs to be loosened I think. Maybe we can make a separate gajim-addons.profile to resolve it?

Reply:
Until now the most plugins in gajim where supported. If you want to create a gajim-addons-common.proile/.inc, go for it.

Theres a differences between addons in e.g. firefox and gajim or eog for example. 

Gajim (and eog and some other) have "core-plugins". Some features are implemented via plugins.

Reply:
I'll see it later, if it is not too much of a change then I'll add them to gajim.profile. The current profile in master won't allow to use gpg because of no access to .gnupg etc.

Reply:
I checked all of the plugins out the most problematic was gpg; added support for it; rest of them didn't need anything extra in the profile. And netlink socket is not needed.